index.html

```diff
@@ -26,14 +26,14 @@
   
     <div class="headline">
         <div><h1 class="text1">I'm Burak <br> Glad to see you</h1></div>
-       <div> <p class="text2">
+       <div> <h2 class="text2">
           I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
           you see and have a project you need coded, don’t hestiate to contact
           me.
-        </p></div>
+       </h2></div>
```

Comment:
- [ ] Instead of using h2 here blush Please use p because this is the paragraph of your page and according to the [heading hierarchy](https://www.w3.org/WAI/test-evaluate/preliminary/#headings) +1